src/solvers/dgsem_tree/subcell_limiters_2d.jl

```diff
@@ -0,0 +1,114 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# this method is used when the limiter is constructed as for shock-capturing volume integrals
+function create_cache(indicator::Type{SubcellLimiterIDP},
+                      equations::AbstractEquations{2},
+                      basis::LobattoLegendreBasis, number_bounds)
+    container_subcell_limiter = Trixi.ContainerSubcellLimiterIDP2D{real(basis)
+                                                                   }(0,
+                                                                     nnodes(basis),
+                                                                     number_bounds)
+
+    cache = (; container_subcell_limiter)
```

Comment:
When changing the name of `container_antidiffusive_fluxes` we should probably change it here too, right?
The name `subcell_limiter` would still be free. But wouldn't it lead to more confusion? @sloede

Reply:
Yes, that's good.
Since later, the local bounds will be saved in there as well, I'll go with `subcell_limiter_coefficients`. It doesn't fit perfectly, including the bounds, but I think that's acceptable.